examples/semantic_version_checker/mkver.conf

```diff
@@ -0,0 +1,34 @@
+tagPrefix: ""
```

Comment:
Great to have this example.

I'd add a second example with a mkver.conf for pre-1.0.0 releases where you can break on minor.

That'll pretty much round out everything!